python/api/action/views.py

```diff
@@ -163,7 +160,7 @@ def get(self, request, *args, **kwargs):
             self.get_queryset(),
             id=action_id,
         )
-        set_disabling_cause(obj, action)
+        set_start_impossible_reason(obj, action)
```

Comment:
Passing parameters only as named agreed. Setter of Action class can be defined instead of separate function